src/schema/types/user.graphql

```diff
@@ -1,4 +1,9 @@
 type User {
   id: ID!
   name: String!
+  groupsSubscribed: [Group]
+}
+
+extend type Query {
+  currentUser: User!
```

Comment:
This should be not required (an anonymous user).

Reply:
Applied!